README.md

```diff
@@ -14,10 +14,52 @@ International Symposium on Performance Analysis of Systems and Software
 ============================================
 Requirements:
 -------------------
-  LLVM 3.4.0 and Clang 3.4.0 64-bit
+  LLVM 3.5 and Clang 3.5 64-bit
+
+Build with CMake:
+-----------------
+  CMake is a configure tool which allows you to out-of-source build. 
+  LLVM-Tracer Requires CMake newer than 2.8.8
+
+  1. Set `LLVM_HOME` to where you installed LLVM
+     ```
+     export $LLVM_HOME=/path/to/your/llvm/installation
+     ```
+
+  2. configure with CMake and build LLVM-Tracer source code
+     ```
+     cd /path/to/build/LLVM-Tracer/
+     cmake /path/to/LLVM-Tracer/
+     make
+     ```
+
+  3. (Optional) Try running triad example, which is built by CMake.
+     ```
+     cd /path/to/build/LLVM-Tracer/
+     ctest -V
+     ```
+
+  4. Available CMake settings
+     ```
+     -DLLVM_ROOT=/where/your/llvm/install
+       You may denote the path of LLVM to find/install by this option. if
+       this option is not defined, environment variable LLVM_HOME will be
+       used.
+
+     -DAUTOINSTALL=TRUE
```

Comment:
Which version of LLVM is installed in the auto install mode, 3.4 or 3.5? We should make it clear.